test/e2e/persistentips_test.go

```diff
@@ -378,22 +378,15 @@ func vmiWithMultus(namespace string) *kubevirtv1.VirtualMachineInstance {
 	)
 }
 
-func vmiWithPasst(namespace string) *kubevirtv1.VirtualMachineInstance {
-	const (
-		interfaceName        = "passtnet"
-		cloudInitNetworkData = `
-version: 2
-ethernets:
-  eth0:
-    dhcp4: true`
-	)
```

Comment:
for managedTap we need to activate dhcpv4 and dhcpv6.

Reply:
aint kubevirt assign ipv4 by default ? 
VM does get ipv4, and this what i remember for the past, wdyt 

ipv6 we don't test on git actions, (also on OVN we dont, because git actions doesnt support it), do we still want to add it ?

Reply:
wait. This depends on the cluster's configuration.
We need to introspect the cluster, figure out which IP families are enabled, and configure this accordingly. 

OR we can hard-code it to support whatever it is we're deploying by default in the e2e tests, but make a real good note about it, then open an issue to introspect in a follow-up PR.

Reply:
> aint kubevirt assign ipv4 by default ? VM does get ipv4, and this what i remember for the past, wdyt ?
> 
> ipv6 we don't test on git actions, (also on OVN CI we dont have the cluster with ipv6, because git actions doesnt support it), do we still want to add it ? EDIT - hmm this one does have `.github/workflows/test.yml: KIND_IPV6_SUPPORT: "${{ matrix.ipfamily == 'IPv6' || matrix.ipfamily == 'dualstack' }}"`

That's not true, we run dual stack at ovn-k, and is ovn-k the one handling IPs.

Reply:
best imo the 2nd option, have it on a follow-up

Reply:
> wait. This depends on the cluster's configuration. We need to introspect the cluster, figure out which IP families are enabled, and configure this accordingly.
> 
> OR we can hard-code it to support whatever it is we're deploying by default in the e2e tests, but make a real good note about it, then open an issue to introspect in a follow-up PR.

We just acivate dhcp for both and that's all, if cluster is just ipv4 we will not have an ipv6 address.

Reply:
i dont think we should add code that might not have an effect, better to have a task to investigate as Miguel said, what cluster we run here and only then add whatever config that is needed

Reply:
btw does ipv6 is tested on OVN repo ? also something that worth checking
Once we have agreement on the direction i will do that, personally i prefer the one that doesnt have unused code and to open issue that is focused and changed whatever needed on all repos.
We can first finish the IPv4 flow on all the repos and then add the IPv6 tests according needs (and according what cluster we have).
Atm we are running IPv4 cluster on this repo.

Reply:
> btw does ipv6 is tested on OVN repo ?

It surely is in some lanes. It's a configuration knob.

> Atm we are running IPv4 cluster on this repo.

So we'll need to change that. That means we're testing half the feature.

Sure, for now, you can move forward with ipv4 (since it appears it's the only thing we support).

But ensure you open a ticket to add IPv6 support to this repo.

Reply:
> It surely is in some lanes. It's a configuration knob.

ye i meant if it is tested for managedTap itself on OVN

np will open Jira ticket
thanks